File: app/src/main/java/com/example/calculoborracha/util/Valores.java
package com.example.calculoborracha.util;

import java.text.DecimalFormat;
import java.text.NumberFormat;

public class Valores {

    public String pesoFormatado(double peso ) {

        DecimalFormat padraoQuilos = new DecimalFormat("###,###,###" );
        return padraoQuilos.format( peso ).replace(",", ".");

    };


    public String valorFormatado(double valor ) {

        DecimalFormat padraoMonetario = new DecimalFormat("" );
        return padraoMonetario.format(valor).replace(",", ".");
//        NumberFormat padraoMonetario = NumberFormat.getIntegerInstance();
//        return padraoMonetario.format(valor);
    }

    public String valorFormatado2(double valor ) {

//        DecimalFormat padraoMonetario = new DecimalFormat("" );
//        return padraoMonetario.format(valor).replace(",", ".");
        NumberFormat padraoMonetario = NumberFormat.getIntegerInstance();
        return padraoMonetario.format(valor);


    }
}

File: app/src/main/java/com/example/calculoborracha/util/Valores.kt
package com.example.calculoborracha.util

import java.text.DecimalFormat
import java.text.NumberFormat

class Valores {
    fun pesoFormatado(peso: Double): String {
        val padraoQuilos = DecimalFormat("###,###,###")
        return padraoQuilos.format(peso).replace(",", ".")
    }

    fun valorFormatado(valor: Double): String {
        val padraoMonetario = DecimalFormat("")
        return padraoMonetario.format(valor).replace(",", ".")
        //        NumberFormat padraoMonetario = NumberFormat.getIntegerInstance();
//        return padraoMonetario.format(valor);
    }

    fun valorFormatado2(valor: Double): String {

//        DecimalFormat padraoMonetario = new DecimalFormat("" );
//        return padraoMonetario.format(valor).replace(",", ".");
        val padraoMonetario = NumberFormat.getIntegerInstance()
        return padraoMonetario.format(valor)
    }
}